jerry-core/rcs/rcs-chunked-list.cpp

```diff
@@ -15,208 +15,304 @@
 
 #include "rcs-chunked-list.h"
 
+
+
+/**
+ * Set previous node for the specified node
+ */
+static void
+rcs_chunked_list_set_prev (rcs_chunked_list_node_t *node_p, /**< node to set previous for */
+                              rcs_chunked_list_node_t *prev_node_p) /**< the previous node */
+{
+  JERRY_ASSERT (node_p != NULL);
+
+  MEM_CP_SET_POINTER (node_p->prev_cp, prev_node_p);
+}
```

Comment:
Usually, we place comment with function name after closing bracket, related to the function.
In the case, the line should be the following: `} /* rcs_chunked_list_set_prev */`.